src/main/java/com/Model/User.java

```diff
@@ -0,0 +1,68 @@
+package com.Model;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String emailAddress;
+    private String password;
+    private boolean isEnabled; //False if user has not confirmed email address yet
+    private boolean isAdmin; //bu default create normal users (not admin)
```

Comment:
Maybe make Admin a subclass of User so you don't have to manage the boolean? In the DB tables the boolean will be created for you.